lib/plausible_web/components/generic.ex

```diff
@@ -865,26 +865,59 @@ defmodule PlausibleWeb.Components.Generic do
   end
 
   attr :href, :string, default: nil
-  attr :icon, :atom, default: :pencil_square
+  attr :class, :any, default: nil
```

Comment:
Ditto about alternative:

```suggestion
  attr :class, :string, default: ""
```